src/Tribe/Views/V2/Views/Widgets/Widget_View.php

```diff
@@ -61,6 +62,37 @@ protected function setup_template_vars() {
 		return $template_vars;
 	}
 
+	/**
+	 * Sets up the View repository arguments from the View context or a provided Context object.
+	 *
+	 * @since 4.9.3
+	 *
+	 * @param  Context|null $context A context to use to setup the args, or `null` to use the View Context.
+	 *
+	 * @return array The arguments, ready to be set on the View repository instance.
+	 */
+	protected function setup_repository_args( Context $context = null ) {
+		bdump('widget_setup_repository_args');
+		$context = null !== $context ? $context : $this->context;
+		$context_arr = $context->to_array();
+		$args        = parent::setup_repository_args( $context );
+
+		/**
+		 * A widget-specific filter for repository args, based on widget slug.
+		 * Allows other plugins to add/remove args for the repository pre-query.
+		 *
+		 * @param $args
+		 * @param $context
+		 */
+		$args = apply_filters( "tribe_events_views_v2_{$this->get_slug()}_repository_args", $args, $context, $context_arr );
+		bdump( [
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: ebaeb7952e6686f114ebd36837ee0b0a</sub>